src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectConfigurationStateSynchronizer.cs

```diff
@@ -73,21 +73,21 @@ public void ProjectConfigurationFileChanged(ProjectConfigurationFileChangeEventA
                         // We found the last associated project file for the configuration file. Reset the project since we can't
                         // accurately determine its configurations.
 
-                        EnqueueUpdateProject(lastAssociatedProjectKey, projectInfo: null);
+                        EnqueueUpdateProject(lastAssociatedProjectKey, projectInfo: null, CancellationToken.None);
                         return;
                     }
 
                     if (!_configurationToProjectMap.TryGetValue(configurationFilePath, out var associatedProjectKey))
                     {
                         _logger.LogWarning("Found no project key for configuration file. Assuming new project. Configuration file path: '{0}'", configurationFilePath);
 
-                        AddProject(configurationFilePath, projectInfo);
+                        AddProjectAsync(configurationFilePath, projectInfo, CancellationToken.None).Forget();
```

Comment:
If the task throws an exception do we still report it? I assume so, but want to make sure

Reply:
I think it depends on where the error is thrown. I'll add a try/catch and log any errors.